app/Application.js

```diff
@@ -29,6 +29,10 @@ Ext.define('momo.Application', {
         var moduleUtil = momo.util.Module;
         var urlUtil = momo.util.URL;
 
+        // disable annoying debug messages from WAI-ARIA 1.0 recommendations.
+        Ext.enableAriaButtons = false;
+        Ext.enableAriaPanels = false;
```

Comment:
I am not sure, I generally strive for high accessibility. If this simply warns without much we can actually do to improve A11Y, then I am plus one. You decide @annarieger